server/bin/pbench-audit-server.sh

```diff
@@ -1,783 +0,0 @@
-#! /bin/bash
-
-# Audit the fs-version-001 archive, incoming, results, and users
-# directory structures.
```

Comment:
I had considered that, but the code is rotting as it stands, and since I am removing the other interfaces which removes the rotting test sub-system, there is no point in leaving this method without a way to verify it still works.